packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -167,13 +166,12 @@ export function TransferPanel() {
 
   const latestDestinationAddress = useLatest(destinationAddress)
 
+  const [dialogProps, openDialog] = useNewDialog()
```

Comment:
It's called `useNewDialog` for now because it will eventually replace the existing dialog and be called `useDialog`. 

We can temporarily name it something else we want, open to suggestions

Reply:
Yea I like it more, updated